### PR TITLE
Adds a positive moodlet to seeing a dog playing

### PR DESCRIPTION
**Pull request by stinky-lizard**
## About The Pull Request

Adds a small positive moodlet (intensity = 2, timeout = 3 minutes) whenever one sees a dog playing ("chases its tail", "dances around"). For instance: "Ian dances around!" -> "Aww, Ian's having fun!"

## Why It's Good For The Game

It's cute. Personally I feel happy whenever my pet is having fun. Also, another positive moodlet (however small) will offset the chaos on station, and dogs aren't all that common on station. There's usually only, what, 3? And they're usually dead, so they can't play.

## Changelog
:cl:
add: You now get a positive moodlet whenever you see a dog playing.
/:cl:

### CODE REVIEW

File: code/datums/ai/dog/dog_controller.dm [hunk 1]
```diff
@@ -75,6 +75,9 @@
 	else if(DT_PROB(10, delta_time))
 		living_pawn.manual_emote(pick("dances around.", "chases [living_pawn.p_their()] tail!</span>"))
 		living_pawn.AddComponent(/datum/component/spinny)
+		for (var/mob/i in oviewers(living_pawn))
+			if (ishuman(i) && i.client) //if viewer is human and a player. maybe check if isn't SSD? also, do non-human players have moods?
```
Comment (Archanial) [inline code/datums/ai/dog/dog_controller.dm]: check for mind not client

Comment (stinky-lizard) [reply code/datums/ai/dog/dog_controller.dm]: Oh, okay. I'm new to SS13 code, so I didn't know about mind. Does the new commit work?

Comment (Archanial) [reply code/datums/ai/dog/dog_controller.dm]: Yes

**Review by Archanial — DISMISSED** [review]
just so checks run

File: code/datums/ai/dog/dog_controller.dm [hunk 2]
```diff
@@ -75,6 +75,9 @@
 	else if(DT_PROB(10, delta_time))
 		living_pawn.manual_emote(pick("dances around.", "chases [living_pawn.p_their()] tail!</span>"))
 		living_pawn.AddComponent(/datum/component/spinny)
+		for (var/mob/i in oviewers(living_pawn))
+			if (ishuman(i) && i.mind) //if viewer is human and a player. do non-human players have moods?
```
Comment (Archanial) [inline code/datums/ai/dog/dog_controller.dm]:
```suggestion
		for(var/mob/living/carbon/human/H in oviewers(living_pawn))
			if(H.mind)
```
All mob/living can have mind, what you're doing here is giving the moodlet to only humans
Oh and that for look has istype built into it

**Review by Archanial — APPROVED** [review]

**Pull request merged**